pkg/cmd/cli/cmd/create.go

```diff
@@ -110,19 +108,15 @@ func (o *CmdOptions) RunCreate(out io.Writer, cmd *cobra.Command, args []string)
 	if err != nil {
 		// Since we create the master and workers, try to clean up
 		deleteCluster(o)
-		return fmt.Errorf(masterSrvMsg, err)
+		return ErrorString(masterSrvMsg, err)
 	}
 
 	// Note, if spark webui service fails for some reason we can live without it
 	// TODO ties into cluster status, make a note if the service is missing
 	sc.Create(&websv.Service)
 
-	if _, err := fmt.Fprintf(out, "cluster \"%s\" created \n",
```

Comment:
Since in this case we return after every individual error, those no reason to accumulate errors in allErrs. We just simply return a nil if we make it to the end.